preview-src/page-templates/integrations-detail.adoc

```diff
@@ -0,0 +1,140 @@
+= LangChain
+:page-layout: tutorial
+// :page-icon: /img/langchain.svg
```

Comment:
I had to comment this out because it was throwing a gulp error. Is it possible to update the preview to support the integration icons?

Reply:
> So the issue is that my svg macros don't work in the preview. It's because the macros use the content catalog, but in the preview that doesn't really seem to be available unfortunately

I assume the `:page-layout:` attribute has the same dependency?